src/components/themed/FilledTextInput.tsx

```diff
@@ -312,14 +313,14 @@ const Container = styled(Animated.View)<{
       borderWidth: theme.textInputBorderWidth,
       borderRadius: theme.rem(0.5),
       flexDirection: 'row',
-      paddingHorizontal: theme.rem(1)
+      paddingHorizontal: theme.rem(0.4)
     },
     useAnimatedStyle(() => ({
       backgroundColor: interpolateInputBackgroundColor(focusAnimation, disableAnimation),
       borderColor: interpolateOutlineColor(focusAnimation, disableAnimation),
       opacity: interpolate(scale.value, [1, 0.5], [1, 0]),
       marginHorizontal: interpolate(scale.value, [1, 0], [0, 2 * rem]),
-      paddingVertical: scale.value * 0.8 * rem
+      paddingVertical: scale.value * 0.75 * rem
```

Comment:
I feel like we would need the extra vertical space to make it feel less crammed.

Reply:
Let's go with .25 rem increments on all our designs. So, change SimpleTextInput to follow this design guideline.